README.md

```diff
@@ -1,24 +1,80 @@
-# Rust binding to [Open Quantum Safe][oqs]'s [liboqs][]
+liboqs-rust: Rust bindings for liboqs
+=====================================
 
-This crate provides bindings to [liboqs][]. It provides unsafe `ffi` bindings in the `oqs-sys` crate.
-Safe wrappers are offered via the `oqs` crate.
+[![Build status](https://github.com/dstebila/oqs-rs/workflows/Continuous%20integration/badge.svg)](https://github.com/dstebila/oqs-rs/actions?query=workflow%3A"Continuous+integration")
 
+**liboqs-rust** offers a Rust wrapper for the [Open Quantum Safe](https://openquantumsafe.org/) [liboqs](https://github.com/open-quantum-safe/liboqs/) C library, which is a C library for quantum-resistant cryptographic algorithms.
 
-## Adding new algorithms
+Pre-requisites
+--------------
 
-### KEMs:
+liboqs-rust depends on the [liboqs](https://github.com/open-quantum-safe/liboqs) C library; liboqs must first be compiled as a Linux/macOS/Windows library (i.e. using `ninja install` with `-DBUILD_SHARED_LIBS=ON` during configuration), see the specific platform building instructions below.
+
+<span style="color: red;">TODO: Check?</span>
+
+Contents
+--------
+
+This crate provides unsafe `ffi` bindings in the `oqs-sys` crate, and safe wrappers are offered via the `oqs` crate.
+
+Usage
+-----
+
+<span style="color: red;">TODO</span>
```

Comment:
This should link to the rustdoc that is generated in the `gh-pages` branch by CI (which can be published as a github website through the github settings).

Reply:
Is there something we can put directly in the README so it's self contained?

Reply:
We can probably copy a bit from the examples listed in the source code and then point to the docs for the rest.